Plugins~/Src/MeshSyncClientBlender/msblenUtils.cpp

```diff
@@ -64,6 +65,18 @@ std::string get_path(const Object *arm, const Bone *obj)
     return ret;
 }
 
+Object* get_object_from_path(std::string path) {
+    auto lastIndexOfDivider = path.find_last_of('/');
+    if (lastIndexOfDivider < 0) {
+        return nullptr;
+    }
+
+    auto objName = path.substr(lastIndexOfDivider + 1);
+
+    bl::BlenderPyScene scene = bl::BlenderPyScene(bl::BlenderPyContext::get().scene());
+    return scene.get_object_by_name(objName);
```

Comment:
Can't Blender have two objects with the same names, but different paths ?

Reply:
No, the path just defines the parenting, the object names are still unique.